app/controllers/quotes_controller.rb

```diff
@@ -43,6 +44,7 @@ def update
       if @quote.update(quote_params)
         format.html { redirect_to @quote, notice: 'Quote was successfully updated.' }
         format.json { render :show, status: :ok, location: @quote }
+        format.turbo_stream
```

Comment:
3.- But I had a problem, the famous "Content missing", and I proceeded to correct it by adding turbo stream in the update action.

![Screenshot 2024-11-07 at 5 16 09 p m](https://github.com/user-attachments/assets/b75ed39b-4095-4772-ae35-91907cc5946e)